pkg/gofr/external_db_test.go

```diff
@@ -0,0 +1,107 @@
+package gofr
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"go.uber.org/mock/gomock"
```

Comment:
both are external imports, can be clubbed together.

Reply:
I'm surprised gci didn't report via golangci-lint 🤔